docs/nlu/training-data-format.rst

```diff
@@ -155,7 +152,6 @@ for these extractors. Currently, all intent classifiers make use of available re
 
 Lookup Tables
 -------------
-Lookup tables in the form of external files or lists of elements may also be specified in the training data.
 The externally supplied lookup tables must be in a newline-separated format.
```

Comment:
We could rephrase this sentence so that it doesn't specify "external" (since this is now implicit):
```suggestion
 The supplied lookup table files must be in a newline-delimited format.
```